app/static/openapi.yaml

```diff
@@ -517,6 +517,94 @@ paths:
         404:
           $ref: '#/components/responses/NotFound'
 
+  /resources/{id}/upvote:
+    put:
+      summary: Upvote Resource - Increment Number of Upvotes
+      description: |
+        Increments the `upvotes` property of a resource. If the user has already upvoted the resource, upvotes property will decrement since a user is limited to only one vote per resource.
```

Comment:
Do you think we should also go ahead and call out that if a user has upvoted a resource and they use the `/downvote` endpoint, it will remove their upvote and replace it with a downvote? Or is that pretty obvious by saying that "a user is limited to one vote per resource"?

Reply:
Yeah. I think it makes more sense to mention that. I'll update the descriptions for upvote and downvote.